packages/taiko-client/internal/docker/nodes/docker-compose.yml

```diff
@@ -16,11 +16,11 @@ services:
       - --hardfork
       - cancun
 
-  l2_execution_engine:
-    container_name: l2_node
+  l2_geth:
+    container_name: l2_geth
     image: us-docker.pkg.dev/evmchain/images/taiko-geth:taiko
     restart: unless-stopped
-    pull_policy: always
+    pull_policy: missing
```

Comment:
why not always pulling the latest image?

Reply:
It's an optional change for faster local debugging.